qutip/tests/test_utilities.py

```diff
@@ -127,19 +127,43 @@ class TestFitting:
     def model(self, x, a, b, c):
         return np.real(a * np.exp(-(b + 1j * c) * x))
 
-    def test_same_model(self):
+    @pytest.fixture
+    def generate_data(self):
```

Comment:
Now that the fitting is moving a bit more in the spotlight, it might be good to test a little better. How about something like

```suggestion
    @pytest.fixture
    @pytest.mark.parametrize("noisy", [False, True])
    def generate_data(self, noisy):
```

and add some small random noise to `y`?

Reply:
Sounds Good!